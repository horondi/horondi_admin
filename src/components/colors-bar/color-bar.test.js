import React from 'react';
import Enzyme, { mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import ColorsBar from './colors-bar';

Enzyme.configure({ adapter: new Adapter() });

describe('should render colors-bar component', () => {
  const colors = [
    {
      color: 'red'
    },
    {
      color: 'white'
    }
  ];
  const fu = () => {
    const a = 5;
    return a;
  };

  const store = {
    foo: 'Foo',
    bar: 'Bar'
  };

  let component;

  beforeEach(() => {
    component = mount(<ColorsBar onColorChange={fu} colors={colors} />, store);
  });

  it('should exist', () => {
    // expect(component).toMatchSnapshot();
    expect(component.exists());
  });

  it('should have props', () => {
    expect(component.props().onColorChange).toBeDefined();
    expect(component.props().colors).toBeDefined();
  });
});
