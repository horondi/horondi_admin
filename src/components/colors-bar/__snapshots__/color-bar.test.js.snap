// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render colors-bar component should exist 1`] = `undefined`;
